truss-chains/truss_chains/model_skeleton.py

```diff
@@ -24,15 +28,18 @@ def __init__(
                 config["model_metadata"][definitions.TRUSS_CONFIG_CHAINS_KEY]
             )
         )
-
-        override_chainlet_to_service_metadata(truss_metadata.chainlet_to_service)
+        deployment_environment: Optional[definitions.Environment] = (
+            utils.get_deployment_environment(environment)
```

Comment:
I feel like this function is not doing enough to be extra defined in a separate file... Why not just in-line the parsing of the pydantic model.

Reply:
no strong preference here - having this in `utils` allows for some unit testing which is nice but i can go either way here